src/launcher/index.css

```diff
@@ -0,0 +1,137 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+
+.jp-Launcher {
```

Comment:
If you get rid of the `.jp-Launcher-dialog` div, most of its rules can be merged here, including the background color.